### PR TITLE
Rename 'en_jp' to 'ja_en'

### PR DESCRIPTION
**Pull request by vevix**
Related Issue #708.

### CODE REVIEW

File: server/db/migrate/20160527090120_rename_title_key.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      UPDATE franchises
+      SET titles = titles || hstore('ja_en', titles -> 'en_jp'), canonical_title = 'ja_en'
+    SQL
+    # Delete the old :en_jp key
+    execute <<-SQL
+      UPDATE franchises
+      SET titles = delete(titles, 'en_jp')
+    SQL
+    # Update canonical_title default
+    change_column_default :franchises, :canonical_title, 'ja_en'
```
Comment (NuckChorris) [inline server/db/migrate/20160527090120_rename_title_key.rb]: Probably also need to `.where(canonical_title: 'en_jp').update(canonical_title: 'ja_en')` (which means creating a migration-internal `class Franchise < ActiveRecord::Base; end` for best-practice reasons)

File: server/db/migrate/20160527090120_rename_title_key.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class RenameTitleKey < ActiveRecord::Migration
+  def change
+    # Copy values from :en_jp to new key :ja_en, update canonical_title values
+    execute <<-SQL
+      UPDATE franchises
```
Comment (NuckChorris) [inline server/db/migrate/20160527090120_rename_title_key.rb]: We probably need to do the same for `anime`, `manga`, and `drama` tables

File: server/db/migrate/20160527090120_rename_title_key.rb (hunk 1 above)
Comment (vevix) [inline server/db/migrate/20160527090120_rename_title_key.rb]: I did an update on canonical_title in the first UPDATE statement. Seemed like the field was added in the previous migration so I assumed all fields would still be the default (`en_jp`).

File: server/db/migrate/20160527090120_rename_title_key.rb (hunk 2 above)
Comment (vevix) [inline server/db/migrate/20160527090120_rename_title_key.rb]: Aren't they actually using `ja_en` already? I'll have to confirm.

Comment (NuckChorris) [inline server/db/migrate/20160527090120_rename_title_key.rb]: wait really, so I didn't fuck up everywhere? :open_mouth: Astonishing!

**Comment by NuckChorris** [comment]
Shit, I have this all backwards.  The proper is actually `en_JP`, following the format `ll_CC`

https://www.w3.org/International/questions/qa-choosing-language-tags

**Comment by vevix** [comment]
Okay will try get around to this tonight!

**Comment by NuckChorris** [comment]
Could you add `episodes` to the list?  `franchises` already has the proper `en_jp`, so `episodes` should be the only remaining table which uses `ja_en`

**Comment by vevix** [comment]
Sure,  missed that one

**Comment by NuckChorris** [comment]
Awesome, other than that lgtm

**Comment by vevix** [comment]
@NuckChorris Can you take a quick look over this again? Few more changes to keys throughout the app.

**Comment by NuckChorris** [comment]
Looks solid to me :tada:

**Pull request merged**